sys/conf/kern.opts.mk

```diff
@@ -99,9 +99,11 @@ __DEFAULT_NO_OPTIONS += FDT
 __DEFAULT_YES_OPTIONS += FDT
 .endif
 
-# Broken post OpenZFS import
 .if ${MACHINE_CPU:Mcheri}
+# Broken post OpenZFS import
```

Comment:
This comment at least is no longer true, but that's separate from this commit (and we are now building OpenZFS by default, so presumably CDDL isn't controlling that anymore?)

Reply:
CDDL is really just DTRACE (plus CTF for us downstream), ZFS is its own independent knob these days... bit confusing

Reply:
I would appreciate having DTrace build sooner so that as I do upstream merges I keep it building and have the functional changes come in later.

Reply:
Does it fail gracefully when run?

Reply:
No, there's lots of ways to panic or wedge the machine :) But it'd be easy to prevent one from loading dtrace.ko if that's a concern.

Reply:
Having said that so long as it stays on dev it's probably fine; just need to make sure it doesn't make it into a release like that

Reply:
Yeah, I think merging it into `dev` now is fine so long as it gets fully finished before the next release (or we put a gratuitous return (ENXIO) in the MOD_LOAD handler right before the release)